src/main/java/games/strategy/engine/framework/startup/launcher/GameLauncher.java

```diff
@@ -0,0 +1,54 @@
+package games.strategy.engine.framework.startup.launcher;
+
+import java.awt.Component;
+
+import javax.swing.JOptionPane;
+import javax.swing.SwingUtilities;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.framework.startup.mc.GameSelectorModel;
+import games.strategy.engine.framework.ui.background.WaitWindow;
+
+
+
+/**
+ * Abstract class for launching a game.
+ */
+public abstract class GameLauncher {
```

Comment:
Looks like this class was copied from `AbstractLauncher`.  I can't find any references to either `AbstractLauncher` nor `ILauncher`.  Can those types be removed?

Reply:
indeed they can be, added that to this branch in: f711787